src/scripts/train/OLMo2-32B.py

```diff
@@ -85,10 +89,57 @@ def build_trainer_config(common: CommonComponents) -> TrainerConfig:
             WandBCallback(
                 name=common.run_name,
                 entity="ai2-llm",
-                project="OLMo-core-26B",
+                project=project_name,
                 enabled=False,
                 cancel_check_interval=10,
             ),
+        ).with_callback(
```

Comment:
We should just add this to the common callbacks.
https://github.com/allenai/OLMo-core/blob/5de774f3046b21e3e4619426f4de5d460367a3e4/src/olmo_core/internal/experiment.py#L163

Reply:
I don't know that we want these for everything. Default should probably be only the new, blessed ones.